BlueprintUI/Sources/BlueprintView/BlueprintView.swift

```diff
@@ -546,6 +548,12 @@ public struct BlueprintViewRenderMetrics {
 
     /// The time it took to update the on-screen views for the element.
     public var viewUpdateDuration: TimeInterval
+
+    /// The mach time in seconds at which the view render started (from `CACurrentMediaTime()`).
+    public var startTimestamp: TimeInterval
+
+    /// The mach time in seconds at which the view finished updating (from `CACurrentMediaTime()`).
+    public var endTimestamp: TimeInterval
```

Comment:
Do we need the `endTimestamp` given we have the start and total duration?

Reply:
I thought about that as well - I'm happy to remove it.

Reply:
I'd probably omit it, or just make it a computed property.

Reply:
I removed it!